extensions/api/data-management/transferprocess/build.gradle.kts

```diff
@@ -24,10 +24,12 @@ plugins {
 
 dependencies {
     api(project(":spi:transfer-spi"))
+    api(project(":core:transfer"))
```

Comment:
nit: Modules shouldn't have dependencies to :core:tranfer, but not sure if it's easily solvable in this PR or rather should be a part of separate issue that cleans the dependencies in the whole project